oam/install_scripts/post-install

```diff
@@ -206,24 +206,22 @@ fi
 #setup MariaDB Columnstore system logging
 if [ $user = "root" ]; then
 	$installdir/bin/syslogSetup.sh install > /tmp/syslog_install.log 2>&1
-	chmod 777 -R /dev/shm
 	rm -f /etc/default/columnstore
 else
-	$SUDO chmod 777 /tmp
+	$SUDO chmod 755 /tmp
 	$SUDO rm -fr /tmp/* > /dev/null 2>&1
 	$SUDO $installdir/bin/syslogSetup.sh --installdir=$installdir install > /tmp/syslog_install.log 2>&1
 	$SUDO chown $user:$user $installdir/etc/Columnstore.xml
-	$SUDO chmod -R 777 /dev/shm
+	$SUDO chmod -R 755 /dev/shm
 	$SUDO mkdir /var/lock/subsys > /dev/null 2>&1
-	$SUDO chmod 777 /var/lock/subsys > /dev/null 2>&1
+	$SUDO chmod 755 /var/lock/subsys > /dev/null 2>&1
 	$SUDO rm -f /var/lock/subsys/mysql-Columnstore
-	$SUDO chmod 777 /etc/fstab	
+	$SUDO chmod 755 /etc/fstab	
```

Comment:
we should not be setting the x bit on this file, especially if we are making it world writable. Big security issue.